docs/site/Examples-and-tutorials.md

```diff
@@ -6,34 +6,37 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Examples-and-tutorials.html
 ---
 
-LoopBack 4 comes with the following example projects:
-
-- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
-  Tutorial on setting up a simple hello-world application using LoopBack 4.
+LoopBack 4 comes with the following tutorials:
 
 - **[todo](todo-tutorial.md)**: Tutorial on building a simple application with
   LoopBack 4 key concepts using bottom-up approach.
 
 - **[todo-list](todo-list-tutorial.md)**: Tutorial on introducing related models
   and building their API from the Todo tutorial
 
+- **[soap-calculator](soap-calculator-tutorial.md)**: Tutorial on integrating
+  SOAP web services.
+- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
+  Tutorial on setting up a simple hello-world application using LoopBack 4.
 - **[log-extension](https://github.com/strongloop/loopback-next/tree/master/examples/log-extension)**:
   Tutorial on building a log extension.
 
-- **[rpc-server](https://github.com/strongloop/loopback-next/tree/master/examples/rpc-server)**:
-  An example showing how to implement a made-up RPC protocol.
+LoopBack 4 also comes with example projects, which are available here:
 
-- **[soap-calculator](soap-calculator-tutorial.md)**: Tutorial on integrating
-  SOAP web services.
+- **[Example Projects](Examples.md)**
+
+Projects that were contributed by the LoopBack community are available here:
+
+- **[Contributions from the LoopBack Community](Contributions.md)**
```

Comment:
I think we have a contribution page https://loopback.io/doc/en/contrib/index.html, while I also noticed that https://loopback.io/doc/en/lb4/ doesn't have an explicit menu that links to it, at least on the top level menu. 
I am not sure is it a good idea to add it to this page...I suggest if we want to add an entry for contributions doc, make it a top level menu that links to https://loopback.io/doc/en/contrib/index.html

Reply:
I think the goal was to request Tutorials and Examples from the community for the Contributions page. But we can update the https://loopback.io/doc/en/contrib/index.html page to include loopback-next, because for right now it only applies to loopback 3. Ans then link to it. But I thought we wanted a place specifically for Tuts and Ex.